client/js/socket-events/configuration.js

```diff
@@ -84,4 +84,36 @@ socket.on("configuration", function(data) {
 				$(this).data("lastvalue", nick);
 			});
 	});
+
+	if ($(document.body).hasClass("public")) {
+		const params = new URLSearchParams(document.location.search);
```

Comment:
According to caniuse, this is not available in Edge 16, are we okay with this?

Reply:
Hm. Edge 17 is released with the next Windows update which should be this month.

Reply:
That feels a bit too bleeding-edge, no? As in, `master` will be broken on Edge until the next release, we're not at "last 2 versions" at all anymore lol.
Is there another mechanism we can use for now?

Reply:
OK, I switched back to URIjs.